<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonWidget</class>
 <widget class="QWidget" name="PersonWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="btnName">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QCheckBox" name="chkCredit">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>111</red>
                <green>110</green>
                <blue>108</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="font">
           <font>
            <pointsize>24</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Credit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spnValue">
          <property name="font">
           <font>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="maximum">
           <double>100000000.989999994635582</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnIncrease">
          <property name="font">
           <font>
            <pointsize>40</pointsize>
           </font>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDecrease">
          <property name="font">
           <font>
            <pointsize>40</pointsize>
           </font>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnName</sender>
   <signal>toggled(bool)</signal>
   <receiver>PersonWidget</receiver>
   <slot>button_toggle(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkCredit</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PersonWidget</receiver>
   <slot>value_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnValue</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PersonWidget</receiver>
   <slot>value_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnIncrease</sender>
   <signal>clicked()</signal>
   <receiver>PersonWidget</receiver>
   <slot>increase()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDecrease</sender>
   <signal>clicked()</signal>
   <receiver>PersonWidget</receiver>
   <slot>decrease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>button_toggle(bool)</slot>
  <slot>value_changed()</slot>
  <slot>decrease()</slot>
  <slot>increase()</slot>
 </slots>
</ui>
